Engineering/CodeReviews/Python.md

```diff
@@ -24,5 +24,63 @@ We follow the [PEP 257](https://www.python.org/dev/peps/pep-0257/) Docstring con
 
 Flake8 is an option for teams who want a single wrapper for running Pyflakes and pycodestyle.
 
+#### Integrating automated checks in the application's build pipeline
```

Comment:
I love this!

Another change is proposed which adds a "Secure Coding" heading (#18), and I think to make this useful for that as well, we should create a new heading '## Running Tools' where this example will live, and then when anyone adds a new type of tools, they can update this example.

having great examples like this is what I think CSE needs so we have reusable components that dev crews can just drop into their projects.